custom_components/tuya_local/devices/solar_inverter_stepuptech.yaml

```diff
@@ -0,0 +1,135 @@
+name: Micro Inverter Stepuptech SP800
+products:
+  - id: r73jxzrtcda4uwwb
+    name: SP800
```

Comment:
Move "Stepuptech" into this product specific name, and change the top level name to a generic "Solar inverter" in case other brands also match the config.

Reply:
OK